src/components/state/Process/process.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import './style.css'
+import productivity2 from '../../../resources/img/work-imgs/productivity-1.svg'
+import arrowRight from '../../../resources/img/work-imgs/arrowRight.svg'
+import arrowLeft from '../../../resources/img/work-imgs/arrowLeft.svg'
+class Process extends Component {
+    state = {  }
+    render() { 
+        return ( 
+        <div className = "process">
+            <div className="headers">
+                <h4 className="processMiniHeader">PROCESS</h4>
+                <h1 className = "methodMadnessHeader">The method to our madness</h1>
+                <img src={productivity2} className="processImage"></img>
+                <img src={arrowRight} className = "arrowRight"></img>
+                <img src={arrowLeft} className = "arrowLeft"></img>
+                <h2 className="discovery">01. Discovery</h2>
+                <p className="text">This is our research phase. We try to get as much information about 
```

Comment:
Use Proxima Nova for your text -- you can find the webfonts here: https://github.com/elliakou/proxima-nova-web-fonts

Reply:
Changed it to Montserrat temporarily as was discussed in the Slack convo, will wait for further instruction 👍